e2e/process-services/widgets/multi_line_widget.e2e.ts

```diff
@@ -0,0 +1,112 @@
+
+        let appDefinitions = await alfrescoJsApi.activiti.appsApi.getAppDefinitions();
+        deployedApp = appDefinitions.data.find((currentApp) => {
+            return currentApp.modelId === appModel.id;
+        });
+        process = await appsActions.startProcess(alfrescoJsApi, appModel, app.processName);
+        loginPage.loginToProcessServicesUsingUserModel(processUserModel);
+        done();
+    });
+
+    beforeEach(() => {
+        let urlToNavigateTo = `${TestConfig.adf.url}/activiti/apps/${deployedApp.id}/tasks/`;
+        browser.get(urlToNavigateTo);
+        taskPage.filtersPage().goToFilter(CONSTANTS.TASK_FILTERS.MY_TASKS);
+        taskPage.formFields().checkFormIsDisplayed();
+    });
+
+    afterAll(async (done) => {
+        await alfrescoJsApi.activiti.processApi.deleteProcessInstance(process.id);
+        await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+        await alfrescoJsApi.activiti.adminTenantsApi.deleteTenant(processUserModel.tenantId);
+        done();
+    });
+
+    it('[C268182] Multi-line Text Widget - General Properties', async () => {
+        let label = widget.multilineTextWidget().getFieldLabel(app.FIELD.multiSimple);
+        expect(label).toBe('multiSimple*');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+        let placeHolder = widget.multilineTextWidget().getFieldPlaceHolder(app.FIELD.multiSimple);
+        expect(placeHolder).toBe('Type something...');
+        widget.multilineTextWidget().setValue(app.FIELD.multiSimple, 'TEST');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeFalsy();
+    });
+
+    it('[C268184] Multi-line Text Widget - Advanced Properties - Min and Max', async () => {
+        widget.multilineTextWidget().setValue(app.FIELD.multiMinMax, 'A');
+        expect(widget.multilineTextWidget().getErrorMessage(app.FIELD.multiMinMax)).toBe('Enter at least 4 characters');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+        widget.multilineTextWidget().setValue(app.FIELD.multiMinMax, 'AAAAAAAAAAA');
+        expect(widget.multilineTextWidget().getErrorMessage(app.FIELD.multiMinMax)).toBe('Enter no more than 10 characters');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+    });
+
+    it('[C268184] Multi-line Text Widget - Advanced Properties - Regex Pattern property', async () => {
+        widget.multilineTextWidget().setValue(app.FIELD.multiSimple, 'TEST');
+        widget.multilineTextWidget().setValue(app.FIELD.multiRegexp, '3');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+        expect(widget.multilineTextWidget().getErrorMessage(app.FIELD.multiRegexp)).toBe('Enter a different value');
+        widget.multilineTextWidget().setValue(app.FIELD.multiRegexp, 'TE');
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeFalsy();
+    });
+
```

Comment:
Could you merge the 2 it above into one and name it : Should be able to set advanced properties for Multi-line Text Widget ?